pkgs/by-name/ar/arcan/package.nix

```diff
@@ -201,6 +202,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru = {
     wrapper = callPackage ./wrapper.nix { };
+    updateScript = nix-update-script {};
```

Comment:
I did not like it, to be honest.

Reply:
But it works. Any suggestion?

Reply:
It is just a wrapper to https://github.com/Mic92/nix-update - an external program -, and it is made to be interactive.

A mere `gitUpdater` should suffice.

Reply:
Note that upstream has moved to fossil (instead of git), but for now adding auto-update infrastructure for the git mirros is still fine.

Reply:
I am working on the fossil, indeed.

Reply:
I am working on the fossil, indeed.